src/core/dataset/data_types.ts

```diff
@@ -214,3 +214,37 @@ export function inferAndConvertColumn(
     metadata: { kind: DataKind.Categorical }
   };
 }
+
+export function convertColumnType(values: any[], type: DataType): DataValue[] {
+  switch (type) {
```

Comment:
Where ever you are doing v.toString(), it can be null/under, might want to add a case checking for that.